modules/nw-multus-ipam-object.adoc

```diff
@@ -83,29 +68,70 @@ ifdef::json[]
       "domain": "<domain>", <10>
       "search": ["<search_domain>"] <11>
     }
+  }
 }
 ----
-<1> `type`: {type}
+<1> Specify `static` to configure the plug-in to manage IP address assignment.
+Specify `DHCP` to allow a DHCP server to manage IP address assignment. You
+cannot specify any additional parameters if you specify a value of `DHCP`.
+
+<2> An array of that define IP addresses to assign to the virtual
+interface. Both IPv4 and IPv6 IP addresses are supported.
+
+<3> A block of IP addresses that you specify in CIDR format to assign
+to Pods on a worker node, such as `10.1.1.0/24`.
+
+<4> The default gateway to route egress network traffic to.
+
+<5> An array describing routes to configure inside the Pod.
 
-<2> `addresses`: An array of {addresses}
+<6> The IP address range in CIDR format.
 
-<3> `address`: {address}
+<7> The gateway to use to route network traffic to.
 
-<4> `gateway`: {address-gateway}
+<8> The DNS configuration. Optional.
 
-<5> `routes`: An array {routes}
+<9> An of array of one or more IP addresses for to send DNS queries to.
 
-<6> `dst`: {route}
+<10> The default domain to append to a host name. For example, if the
```

Comment:
s/prepend/append?

Reply:
The way it works, the DNS resolver adds the domain to the end of a host name that isn't _qualified_. So `fedora` is a host name, unqualified. And thus the resolver will append `example.com`, giving us `fedora.example.com` to resolve.

Actually I think it's `fedora.example.com.` or some such, but I've long forgotten what DNS I knew.